apps/timetable/admin/index.html

```diff
@@ -140,11 +140,23 @@
             <%= _.partial('new-module-modal', {'data': data}) %>
         </script>
 
+        <!-- Rename module modal -->
+        <div id="gh-rename-module-modal-container"><!-- --></div>
+        <script id="gh-rename-module-modal-template" type="text/template">
+            <%= _.partial('rename-module-modal', {'data': data}) %>
+        </script>
+
         <!-- Batch edit template -->
         <script id="gh-batch-edit-template" type="text/template">
             <%= _.partial('admin-batch-edit', {'data': data}) %>
         </script>
 
+        <!-- Delete module modal -->
+        <div id="gh-delete-module-modal-container"><!-- --></div>
+        <script id="gh-delete-module-modal-template" type="text/template">
+            <%= _.partial('delete-module-modal', {'data': data}) %>
```

Comment:
We can probably just do `<%= _.partial('delete-module-modal') %> here since we're not passing in data from the view controller.